web-frontend/.env.massastation

```diff
@@ -1 +1,2 @@
-VITE_BASE_PATH="/thyra/plugin/massalabs/wallet/web-app/"
\ No newline at end of file
+VITE_BASE_PATH="/thyra/plugin/massalabs/wallet/web-app/"
+VITE_API_URL="/thyra/plugin/massalabs/wallet/api/"
```

Comment:
I would suggest to keep all that in an object. 
Most of the time you'll import BASE_API|APP but you can import granularly if you need it too.

```js
export const API = {
  verb: "http",
  namespace: "",
  server: "localhost",
  port: "3001",
};

export const APP = {
  verb: "http",
  namespace: "",
  server: "localhost",
  port: "3000",
};

export const BASE_API = API.port
  ? `${API.verb}://${API.server}:${API.port}`
  : `${API.verb}://${API.server}`;

export const BASE_APP = API.port
  ? `${APP.verb}://${APP.server}:${APP.port}`
  : `${APP.verb}://${APP.server}`;

Reply:
I don't understand your comment, we benefit from vite fonctionnality https://vitejs.dev/guide/env-and-mode.html to make the wallet work for both standalone and as a plugin